### PR TITLE
[docs] Update configuration-ini.html file

### PR DESCRIPTION
**Pull request by songgot**
- Remove learning_rate from oprimizer section
- Add learning rate scheduler section
- Remove dataset section
- Add train set, validation set and test set section
- Add many types to layer section
- Add table about type, key, value, default value and description for each layers
- Update configuration file example

Signed-off-by: Hyunil <hyunil46.park@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM
